query/src/servers/http/v1/statement.rs

```diff
@@ -101,17 +104,21 @@ struct HttpQueryState {
 
 // TODO(youngsofun): add a HttpQueryManager in SessionManger to support async query.
 impl HttpQuery {
-    fn new(id: String, sql: String) -> HttpQuery {
+    fn new(id: String, sql: String, db: Option<String>) -> HttpQuery {
         HttpQuery {
             id,
             sql,
             state: None,
+            db,
         }
     }
 
     async fn start(&mut self, session_manager: SessionManagerRef) -> Result<HttpQueryState> {
         let session = session_manager.create_session("http-statement")?;
         let ctx = session.create_context().await?;
+        if self.db.is_some() && !self.db.clone().unwrap().is_empty() {
```

Comment:
is_empty()  should not be checked specially,  just let set_current_database("")  return Err?

Reply:
`/v1/statement/?db=` should be acceptd for default database.